ui/src/testing/stubs/runtime-api-stub.ts

```diff
@@ -8,18 +9,29 @@ import {
 } from 'generated/fetch';
 import {stubNotImplementedError} from 'testing/stubs/stub-utils';
 
+export const mockGceConfig: GceConfig = {
+  diskSize: 80,
+  machineType: 'n1-standard-4'
+};
+
+export const mockDataprocConfig: DataprocConfig = {
+  masterMachineType: 'n1-standard-4',
+  masterDiskSize: 80,
+  workerDiskSize: 40,
+  workerMachineType: 'n1-standard-4',
+  numberOfWorkers: 1,
+  numberOfPreemptibleWorkers: 2,
+  numberOfWorkerLocalSSDs: 0
+};
+
 export const defaultRuntime = () => ({
   runtimeName: 'Runtime Name',
   googleProject: 'Namespace',
   status: RuntimeStatus.Running,
   createdDate: '08/08/2018',
   toolDockerImage: 'broadinstitute/terra-jupyter-aou:1.0.999',
   configurationType: RuntimeConfigurationType.GeneralAnalysis,
-  dataprocConfig: {
-    masterMachineType: 'n1-standard-4',
-    masterDiskSize: 80,
-    numberOfWorkers: 0
-  }
+  gceConfig: mockGceConfig
```

Comment:
I would shallow copy here. I made `defaultRuntime` a function to avoid tests interfering with one another by mutating the runtime value.